code/game/machinery/vending.dm

```diff
@@ -263,7 +258,7 @@ var/global/num_vending_terminals = 1
 			else
 				to_chat(user, "<span class='warning'>This recharge pack isn't meant for this kind of vending machines.</span>")
 
-/obj/machinery/vending/proc/normal_refill(obj/structure/vendomatpack/P, mob/user)
+/obj/machinery/vending/proc/normal_refill(obj/structure/vendomatpack/P, mob/user) //TODO: This is totally fucking broken
```

Comment:
:thinking:

Reply:
Oh yeah by this I meant that it's really abusable but it would be unatomic to fix it in a PR about custom vendors